var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -51,6 +54,8 @@ class Paraview(CMakePackage):
     variant('opengl2', default=True, description='Enable OpenGL2 backend')
     variant('examples', default=False, description="Build examples")
     variant('hdf5', default=False, description="Use external HDF5")
+    variant('netcdf', default=False, description="Use external NETCDF")
+    variant('expat', default=False, description="Use external EXPAT")
```

Comment:
Same question for these variants, Spack should always be using external libraries, it should never use internal libraries. Otherwise you'll get multiple copies of the same library in the DAG.

Reply:
That would actually help a fair bit then. Make expat always be external.